sdk/nodejs/ec2/subnet.ts

```diff
@@ -101,6 +108,11 @@ export class Subnet extends pulumi.CustomResource {
  * Input properties used for looking up and filtering Subnet resources.
  */
 export interface SubnetState {
+    /**
+     * The ARN of the subnet.
+     * * `availability_zone`- The AZ for the subnet.
```

Comment:
Bit of an odd comment here... I wonder if pulumi/pulumi-terraform#235 would help?

Reply:
Yeah - hopefully it will - though I suspect there is further work needed as well in general to improve this doc text extraction.